### DISCUSSION TITLE
option for  curators to copy the text  for documents that are not Crawlable

### DISCUSSION THREAD

**Discussion by pederdm000** [tosdr/edit.tosdr.org#1077] [Ideas]
have a way for curators to copy the text for documents and then add them as if the crawler did basically a way to manually add documents text and still allow people to annotate the document. It would also be nice if a second curator had to approve the manually add a document for accuracy. I think we should also have some sort of stats indicator showing weather the text is from a crawler of if it was manually added.

**Comment by elvey** [comment]
I second this.  

Because of https://central.mychart.org/MyChart/app/welcome-wizard/welcome / https://central.mychart.org/MyChart/app/welcome-wizard/terms-and-conditions for example - or maybe a bot can be taught to navigate to the TOS in this case.  **Actually, never mind** because for this case, a bot can handle it - it is crawlable after all.  At first I thought a bot couldn't see the terms, as without the parameters (redirectrequest parameter), the ToS don't load, but with the full URL, they do.
